sqlx-core/src/postgres/types/money.rs

```diff
@@ -35,6 +35,29 @@ impl PgMoney {
 
         bigdecimal::BigDecimal::new(digits, scale)
     }
+
+    /// Convert the money value into a [`Decimal`] using the correct precision
+    /// defined in the PostgreSQL settings. The default precision is two.
+    ///
+    /// [`Decimal`]: ../../types/struct.BigDecimal.html
+    #[cfg(feature = "decimal")]
+    pub fn to_decimal(self, scale: u32) -> rust_decimal::Decimal {
+        rust_decimal::Decimal::new(self.0, scale)
+    }
+
+    /// Convert a [`Decimal`] value into money using the correct precision
+    /// defined in the PostgreSQL settings. The default precision is two.
+    ///
+    /// [`Decimal`]: ../../types/struct.BigDecimal.html
```

Comment:
We like using intra-doc links: https://github.com/rust-lang/rfcs/blob/master/text/1946-intra-rustdoc-links.md#additions-to-the-documentation-syntax

It also looks like this is referring to the `BigDecimal` docs instead of `Decimal`.